sys/log/full/src/log.c

```diff
@@ -62,7 +62,7 @@ log_init(void)
     log_inited = 1;
     log_written = 0;
 
-    g_log_info.li_version = LOG_VERSION_V2;
+    g_log_info.li_version = LOG_VERSION_V3;
```

Comment:
Should this be:
```
g_log_info.li_version = MYNEWT_VAL(LOG_VERSION);
```
?

Actually, I am not sure I understand why we even need `g_log_info.li_version`.  Can we just use `MYNEWT_VAL(LOG_VERSION)` instead?  I might have missed something in the code, so feel free to correct me :).

Reply:
correct, should be assigned from `MYNEWT_VAL`. will fix.

`g_log_info.li_version` is used in log_fcb and log_reboot to initialize fcb version but obviously this can also be just `MYNEWT_VAL(LOG_VERSION)`. since `g_log_info` is a global struct so I kept it since someone could use it - but tbh I'm not sure what for, so I doubt anyone would really miss it ;-)